app/javascript/react/screens/App/Overview/screens/PlanWizard/helpers.js

```diff
@@ -0,0 +1,27 @@
+import { map, trim } from 'lodash';
+
+export const createMigrationPlans = (
+  planWizardGeneralStep,
+  planWizardCSVStep
+) => {
+  const planName = planWizardGeneralStep.values.name;
+  const planDescription = planWizardGeneralStep.values.description;
+  const infrastructureMapping =
+    planWizardGeneralStep.values.infrastructure_mapping;
+  const vms = map(planWizardCSVStep.values.csvRows, 'reference').map(trim);
```

Comment:
can someone yank lodash back out pretty please??? 

ES6 gives us map and trim natively now:

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_trim
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_map

Reply:
good catch @jeff-phillips-18

Reply:
@priley86 I will look into that.

In the previous versions of lodash, `map` used to be called `pluck`  - which is what I need here.

This link https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_map does not seem to be referring to the `map` (aka `pluck`) that I'm using here.
Maybe it goes by a different name in ES6?